cypress/e2e/hub/hub-ee-access.cy.ts

```diff
@@ -114,16 +110,12 @@ describe.skip('Execution Environment User Access tab', () => {
     cy.intercept('POST', hubAPI`/_ui/v2/role_team_assignments/`).as('teamRoleAssignment');
     cy.createHubTeam().then((hubTeam) => {
       cy.clickTab('Team Access', true);
-      cy.url().should('include', '/team-access');
```

Comment:
Is that one not longer true?

Reply:
it is true, however, I removed it as the goal is to test the addition of the team to the Remotes. URL checks can be done to assert while something on the wizard is still loading